src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/repository/db/UserAuditRepository.java

```diff
@@ -3,7 +3,13 @@
 import java.util.UUID;
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
+import uk.gov.ons.ctp.integration.contactcentresvc.model.AuditType;
 import uk.gov.ons.ctp.integration.contactcentresvc.model.UserAudit;
 
 @Repository
-public interface UserAuditRepository extends JpaRepository<UserAudit, UUID> {}
+public interface UserAuditRepository extends JpaRepository<UserAudit, UUID> {
+
+
+    int countAllByAndCcuserIdAndAuditType(UUID ccUserId, AuditType auditType);
```

Comment:
Does the name need to start with 'countAllByAnd' ?
The 'And' part makes the method name very awkward to read.